python/ray/tune/experiment.py

```diff
@@ -43,6 +43,8 @@ class Experiment(object):
             to (e.g. ``s3://bucket``).
         checkpoint_freq (int): How many training iterations between
             checkpoints. A value of 0 (default) disables checkpointing.
+        checkpoint_at_end (bool): Whether to checkpoint at the end of the
+            experiment. Default is False.
```

Comment:
Can you also note that this is only enabled if checkpoint_freq is not 0?

Reply:
You mean is enabled regardless right?

Reply:
Currently, it will be disabled if checkpoint_freq is 0 as Richard pointed out. Do we want to override and enable this even when checkpoint_freq is 0?

Reply:
I think it will be better to have the checkpoint_at_end option have an effect even if checkpoint_freq is set to 0/default. This will provide more flexibility for the users depending on the use cases. What do you guys think?

Reply:
The new change looks good!